src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -174,9 +174,10 @@ export class PatientViewPageStore {
         return fetchMyCancerGenomeData();
     }
 
-    mutationalSignatureData(): IMutationalSignature[]{
-        return fetchMutationalSignatureData();
-    }
+    readonly mutationalSignatureData = remoteData({
+        invoke: async() => fetchMutationalSignatureData(),
+        default: fetchMutationalSignatureData()

```

Comment:
default isnt necessary here. see comment above for how to use it without default